File: letter/ru/А.py
from common import *

expr = Eq(
    (y + 4 * Abs(x) - 1) *
    (y + S(1) / 4) *
    sqrt(S(1 + eps - y - 4 * Abs(x)) / (1 - Abs(y))),
    0)
size = 1

File: letter/ru/А.js
import { Letter } from '../../Letter.js';

const expr = '(y + 4*abs(x) - 1) * (y + 1/4) * sqrt((1 + 1/10 - y - 4*abs(x))/(1 - abs(y)))';

export const letter = new Letter(expr, 1);
